src/main/java/mekanism/client/ClientTickHandler.java

```diff
@@ -86,15 +87,13 @@ public static boolean isJetpackActive(Player player) {
             return Mekanism.playerState.isJetpackOn(player);
         }
         if (!player.isSpectator()) {
-            ItemStack chest = ItemJetpack.getJetpack(player);
-            if (!chest.isEmpty()) {
-                JetpackMode mode = CommonPlayerTickHandler.getJetpackMode(chest);
+            ItemStack activeJetpack = IJetpackItem.getActiveJetpack(player);
+            if (!activeJetpack.isEmpty()) {
+                JetpackMode mode = ((IJetpackItem) activeJetpack.getItem()).getJetpackMode(activeJetpack);
                 if (mode == JetpackMode.NORMAL) {
                     return minecraft.screen == null && minecraft.player.input.jumping;
                 } else if (mode == JetpackMode.HOVER) {
-                    boolean ascending = minecraft.player.input.jumping;
-                    boolean descending = minecraft.player.input.shiftKeyDown;
-                    if (!ascending || descending || minecraft.screen != null) {
+                    if (!minecraft.player.input.jumping || minecraft.player.input.shiftKeyDown || minecraft.screen != null) {
```

Comment:
I changed it as it wasn't doing that for the block below, but I'll revert and maybe add it to that one

Reply:
Main reason I didn't bother checking it presuming you mean for normal mode that is mainly because it shows what it corresponds to two lines below (where that variable is declared).

Reply:
You're right, for some reason I thought the normal block was below, not above.